train_redpajama.py

```diff
@@ -266,9 +267,8 @@ def create_dataloaders(
 def get_batch(
```

Comment:
now that there are only two lines in this function, should I inline that code in our training loop?

Reply:
yes let's do it

Reply:
done